frontend/src/views/Join.tsx

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { PrimaryButton } from '../components/core/Button';
-import { connect, disconnect } from '../api/Socket';
+import Input from '../components/core/Input';
+import { connect, disconnect, sendUser } from '../api/Socket';
+
+// Create constants for the subscription and send message URLs.
+const SOCKET_ENDPOINT:string = '/api/v1/socket/join-room-endpoint';
```

Comment:
Could this go in the `Socket.ts` file to have a centralized hub for URL paths? That'd mean an extra import and possible style questions for exporting constants so we can also revisit later.

Reply:
No, I think that makes sense; I added the following line to `Socket.ts`:
```
const SOCKET_ENDPOINT:string = '/api/v1/socket/join-room-endpoint';
```
I also added the relevant `import` to `Join.tsx` (this file).